index.html

```diff
@@ -116,17 +115,18 @@ <h2>Main Program<hr></h2>
                     </div> 
                 </div>
                 <p class="main-down-word">SEE THE WHOLE PROGRAM</p>
+                <button class="joincc">Join CC Global Summit 2015</button>
     </section>
     </section>
     <section class="featured-speakers-heading">
             <div class="featured-speakers-heading">
-                <h5>Featured Speakers</h5>
-                <div id="line"></div>
+                <h5>Featured Speakers<hr></h5>
             </div>
             <div id="featured-speakers"></div>
             <div class="more-mobile">
-                <p>MORE</p>
-                <img src="imgs/more-arrow.png">
+            <button>
+                <p>MORE<img src="icon/arrow_down 1.png"></p>
+            </button>
             </div>
     </section>
```

Comment:
- [ ] Your Speaker renders dynamically and impressively but it is not aligned to the center. Please fix this buggy appearance.
- [ ] Also, as we did to section above, let's remove the `line-height` and `font-weight` of 700 from your speaker details - `p` element.
- [ ] If you check the design closely, it looks like the speaker's job title should be in italics. Please check and fix :pray: 

![image](https://github.com/ishimwepati/Conference-Page/assets/99715640/91665a2e-db0c-4ed0-9055-a213040225df)